### PR TITLE
[ci-visibility] Simplify `jest` Instructions

### PR DESCRIPTION
**Pull request by juan-fernandez**
### What does this PR do?
* Simplify `jest` instructions now that we support `jest-jasmine2` too. 

### Motivation
* Make it easier for onboarding customers to use our `jest` instrumentation. 

### Preview

https://docs-staging.datadoghq.com/juan-fernandez/ci-app-add-jasmine/continuous_integration/setup_tests/javascript/?tab=jestjestcircus#instrument-your-tests

### Additional Notes

---

### Reviewer checklist
- [ ] Review the changed files.
- [ ] Review the URLs listed in the [Preview](#preview) section.
- [ ] Check images for PII
- [ ] Review any mentions of "Contact Datadog support" for internal support documentation.

### CODE REVIEW

**Comment by juan-fernandez** [comment]
⚠️ do not merge yet: there might be a better way to organise `jest-jasmine` / `jest-circus` separation.

**Comment by kayayarai** [comment]
> ⚠️ do not merge yet: there might be a better way to organise `jest-jasmine` / `jest-circus` separation.

😎  I added the DNM label, which you can remove when you're ready for review

**Comment by juan-fernandez** [comment]
> > ⚠️ do not merge yet: there might be a better way to organise `jest-jasmine` / `jest-circus` separation.
> 
> Initial review: looks fine, if repetitive. 😎 I added the DNM label, which you can remove when you're ready for review

thanks! I'll check together with the team to see if we can come up with something better

**Comment by juan-fernandez** [comment]
@kayayarai ok I've simplified the instructions. I hadn't realised we could do this now. It should be good now to review and merged 😄

File: content/en/continuous_integration/setup_tests/javascript.md [hunk 1]
```diff
@@ -81,7 +66,7 @@ require('dd-trace').init({
 module.exports = require('jest-environment-node')
 ```
 
-<div class="alert alert-warning"><strong>Note</strong>: <code>jest-environment-node</code> and <code>jest-environment-jsdom</code> are installed together with <code>jest</code>, so they do not normally appear in your <code>package.json</code>. If you've extracted any of these libraries in your <code>package.json</code>, make sure the installed version is the same as the one of <code>jest</code>.</div>
+<div class="alert alert-warning"><strong>Note</strong>: <code>jest-environment-node</code>, <code>jest-environment-jsdom</code>, <code>jest-jasmine2</code> and <code>jest-circus</code> (as of Jest 27) are installed together with <code>jest</code>, so they do not normally appear in your <code>package.json</code>. If you've extracted any of these libraries in your <code>package.json</code>, make sure the installed versions are the same as the one of <code>jest</code>.</div>
```
Comment (kayayarai) [inline content/en/continuous_integration/setup_tests/javascript.md:69]:
```suggestion
<div class="alert alert-warning"><strong>Note</strong>: <code>jest-environment-node</code>, <code>jest-environment-jsdom</code>, <code>jest-jasmine2</code>, and <code>jest-circus</code> (as of Jest 27) are installed together with <code>jest</code>, so they do not normally appear in your <code>package.json</code>. If you've extracted any of these libraries in your <code>package.json</code>, make sure the installed versions are the same as the one of <code>jest</code>.</div>
```

**Review by kayayarai — APPROVED** [review]

**Pull request merged**